### ISSUE TITLE
gap_fit does not find energies if pbc=false

### ISSUE THREAD

**Issue by imagdau** [libAtoms/QUIP#607]
I am trying to gap_fit on a non-periodic dataset of molecular clusters.

Isolated atom configs are not found at all if pbc=false.

When I set pbc=true and a large box just for the isolated atoms, but not for the rest of configs, only 3 energy values are reported in 'Number of target energies', however, forces are identified correctly.

Setting pbc=true for all configs and large dummy boxes fixes the issue and all energy values are found.

**Comment by imagdau** [comment]
I should have mentioned that gap_fit was obtained by installing quippy-ase with pip, pythong=3.10

**Comment by bernstei** [comment]
Is it possible that the fortran code doesn't support non-periodic cells, so we decided to filter them out?  If so, I think that's a bad idea for gap_fit - it should give an error and stop.

**Comment by gabor1** [comment]
yes I think that's probably right, and your suggestion makes sense.
